config-model/src/main/java/com/yahoo/config/model/deploy/TestProperties.java

```diff
@@ -77,6 +77,7 @@ public class TestProperties implements ModelContext.Properties, ModelContext.Fea
     private List<String> environmentVariables = List.of();
     private boolean avoidRenamingSummaryFeatures = true;
     private boolean experimentalSdParsing = true;
+    private boolean enableBitVectors = true;
```

Comment:
I suggest setting this to _false_ to match the current default.

Reply:
Done